hpx/runtime/threads/executors.hpp

```diff
@@ -10,11 +10,11 @@
 #include <hpx/config.hpp>
 
 #if defined(HPX_HAVE_THREAD_EXECUTORS_COMPATIBILITY)
-#include <hpx/runtime/threads/executors/current_executor.hpp>
-#include <hpx/runtime/threads/executors/default_executor.hpp>
-#include <hpx/runtime/threads/executors/service_executors.hpp>
-#include <hpx/runtime/threads/executors/this_thread_executors.hpp>
-#include <hpx/runtime/threads/executors/embedded_thread_pool_executors.hpp>
+#include <hpx/thread_executors/current_executor.hpp>
+#include <hpx/thread_executors/default_executor.hpp>
+#include <hpx/thread_executors/service_executors.hpp>
+#include <hpx/thread_executors/this_thread_executors.hpp>
+#include <hpx/thread_executors/embedded_thread_pool_executors.hpp>
```

Comment:
I think this whole file (`hpx/runtime/threads/executors.hpp`) can move as well, but it can be replaced by the generated module header. You can then point the compatibility header to `hpx/thread_executors.hpp` instead.

Reply:
Yep!